docs/sources/installation/configuration.md

```diff
@@ -350,7 +350,7 @@ Default is `false`.
 
 ### data_source_proxy_whitelist
 
-Define a white list of allowed ips/domains to use in data sources. Format: `ip_or_domain:port` separated by spaces.
+Define a whitelist of allowed IP addresses or domains to be used as data source URLs with the Grafana data source proxy. Format: `ip_or_domain:port` separated by spaces. PostgreSQL, MySQL, and MSSQL data sources do not use the proxy and are therefore unaffected by this setting.
```

Comment:
I don't think it would be right to say that IP addresses or domains can be used _as_ URLs, but rather in URLs. Ports also have to be specified in the whitelist, so I updated the documentation wrt that.
```suggestion
Define a whitelist of allowed IP addresses or domains, with ports, to be used in data source URLs with the Grafana data source proxy. Format: `ip_or_domain:port` separated by spaces. PostgreSQL, MySQL, and MSSQL data sources do not use the proxy and are therefore unaffected by this setting.
```